financialaid/views_test.py

```diff
@@ -128,24 +131,43 @@ def test_income_validation_user_not_enrolled(self):
         self.program_enrollment.save()
         self.assert_http_status(self.client.post, self.request_url, status.HTTP_400_BAD_REQUEST, data=self.data)
 
-    def test_income_validation_currency_not_usd(self):
+    def test_income_validation_currency_not_usd_gto(self):
         """
-        Tests IncomeValidationView post with a currency that is not USD
+        Tests FinancialAidRequestView post with a currency that is not USD with exchange rate greater than 1
```

Comment:
I'm not sure of the logic here, it's the income which ultimately determines the tier. The exchange rate being less or greater than 1 doesn't matter when you can set the input income to any value. But feel free to leave it since it's just an extra test